main/webapp/modules/core/scripts/index/default-importing-controller/file-selection-panel.js

```diff
@@ -223,7 +224,8 @@ Refine.DefaultImportingController.prototype._renderFileSelectionPanelControlPane
     $('<button>')
     .text($.i18n('core-buttons/deselect'))
     .addClass("button")
-    .appendTo($('<td>').appendTo(tr))
+    .css('margin-left','3px')
+    .appendTo(tr)
```

Comment:
Is it really okay to be appending an element directly inside a `<tr>`? If the problem comes from having the two buttons in separate `<td>` elements, perhaps it would be a bit better to append both elements in the same `<td>`?